src/js/actions/libraries.js

```diff
@@ -145,7 +145,7 @@ define(function (require, exports) {
             currentLayer = currentLayers.first();
 
         if (!currentLibrary || currentLayers.size !== 1 ||
-            !currentLayer || currentLayer.kind !== currentLayer.layerKinds.TEXT) {
+            !currentLayer || currentLayer.isTextLayer()) {
```

Comment:
the good side is that it encapsulates the comparison to avoid duplicate (here is another [checker](https://github.com/adobe-photoshop/spaces-design/pull/1747/files#diff-97064c297468dd4ad7cea8982b6ac504R147)), and slightly improves readability. I couldn't come up with other downsides except for the potentials of many more similar checkers (isVideoLayer, isGroupLayer, etc.), which can be solved by using one consolidated function like Layer#isKind(name)